File: DAL/DbModel/Models/FoodProduct.cs
using DbModel.Enums;
using System.ComponentModel.DataAnnotations.Schema;

namespace DbModel.Models
{
	public class FoodProduct
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string? Description { get; set; }
        public FoodType FoodType { get; set; }
        public int FoodBrandId { get; set; }
        [ForeignKey(nameof(FoodBrandId))]
        public FoodBrand FoodBrand { get; set; }
		public ICollection<FoodProductItem>? Items { get; set; }
    }
}

File: projektsmaczek.client/src/models/FoodProduct.ts
import type { FoodBrand } from "./FoodBrand"
import type { FoodProductItem } from "./FoodProductItem"
import { FoodType } from "./enums/FoodType"

export interface FoodProduct {
    key: number;
    name: string;
    description: string | null;
    foodType: FoodType;
    foodBrand: FoodBrand;
    items: FoodProductItem[] | null;
}
